networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/topics/Eth2TopicHandlerTest.java

```diff
@@ -186,7 +209,8 @@ public void handleMessage_errorWhileProcessing_rejectedExecution() {
             asyncRunner,
             (b, __) -> {
               throw new RejectedExecutionException("No more capacity");
-            });
+            },
+            null);
```

Comment:
these nulls are fairly sub-optimal imo... do we have a Noop or something? it looks like they could be just passing something in rather than null.